==== External Call To User-Supplied Address ====
SWC ID: 107
Severity: Low
Contract: testSend
Function name: doSend(address,uint256)
PC address: 116
Estimated Gas Usage: 2778 - 105528
A call to a user-supplied address is executed.
An external message call to an address specified by the caller is executed. Note that the callee account might contain arbitrary code and could re-enter any function within this contract. Reentering the contract in an intermediate state may lead to unexpected behaviour. Make sure that no state modifications are executed after this call and/or reentrancy guards are in place.
--------------------
In file: /tmp/f8296b34227aaad80a2549342395f2e7.sol:16

 _amountETH}("");
        _to.s

--------------------
Initial State:

Account: [CREATOR], balance: 0x20010400000e96f8, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [SOMEGUY], function: doSend(address,uint256), txdata: 0xa3d7bd43000000000000000000000000deadbeefdeadbeefdeadbeefdeadbeefdeadbeef0000000000000000000000000000000000000000000000000000000000000000, value: 0x0

==== Unchecked return value from external call. ====
SWC ID: 104
Severity: Medium
Contract: testSend
Function name: doSend(address,uint256)
PC address: 116
Estimated Gas Usage: 2778 - 105528
The return value of a message call is not checked.
External calls return a boolean value. If the callee halts with an exception, 'false' is returned and execution continues in the caller. The caller should check whether an exception happened and react accordingly to avoid unexpected behavior. For example it is often desirable to wrap external calls in require() so the transaction is reverted if the call fails.
--------------------
In file: /tmp/f8296b34227aaad80a2549342395f2e7.sol:16

 _amountETH}("");
        _to.s

--------------------
Initial State:

Account: [CREATOR], balance: 0x214100000007be7c, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x80, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [ATTACKER], function: doSend(address,uint256), txdata: 0xa3d7bd4300000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000, value: 0x0

==== Multiple Calls in a Single Transaction ====
SWC ID: 113
Severity: Low
Contract: testSend
Function name: doSend(address,uint256)
PC address: 204
Estimated Gas Usage: 2778 - 105528
Multiple calls are executed in the same transaction.
This call is executed following another call within the same transaction. It is possible that the call never gets executed if a prior call fails permanently. This might be caused intentionally by a malicious callee. If possible, refactor the code such that each transaction only executes one external call or make sure that all callees can be trusted (i.e. theyâ€™re part of your own codebase).
--------------------
In file: /tmp/f8296b34227aaad80a2549342395f2e7.sol:17

ETH);
        _to.tr

--------------------
Initial State:

Account: [CREATOR], balance: 0x21c00000020fddda, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [SOMEGUY], function: doSend(address,uint256), txdata: 0xa3d7bd4300000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000000000000, value: 0x0

==== Unchecked return value from external call. ====
SWC ID: 104
Severity: Medium
Contract: testSend
Function name: doSend(address,uint256)
PC address: 204
Estimated Gas Usage: 2778 - 105528
The return value of a message call is not checked.
External calls return a boolean value. If the callee halts with an exception, 'false' is returned and execution continues in the caller. The caller should check whether an exception happened and react accordingly to avoid unexpected behavior. For example it is often desirable to wrap external calls in require() so the transaction is reverted if the call fails.
--------------------
In file: /tmp/f8296b34227aaad80a2549342395f2e7.sol:17

ETH);
        _to.tr

--------------------
Initial State:

Account: [CREATOR], balance: 0x21400000020ffdfa, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x4000000000001, nonce:0, storage:{}
Account: [SOMEGUY], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , value: 0x0
Caller: [ATTACKER], function: doSend(address,uint256), txdata: 0xa3d7bd4300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, value: 0x0